stuck.js

```diff
@@ -120,6 +120,7 @@ new Vue({
 
     data: function() {
         return {
+            show: true,
```

Comment:
The method `setState` should also set `this.show = false` to emulate the previous behavior.